clirr-ignores.xml

```diff
@@ -147,4 +147,12 @@
         <justification>False positive, the enclosing class is package-private so this was never exposed</justification>
     </difference>
 
+    <difference>
+        <differenceType>7005</differenceType>
+        <className>com/datastax/driver/core/querybuilder/QueryBuilder</className>
+        <method>*</method>
```

Comment:
Unfortunately, having such a broad exception is necessary otherwise clirr repeats errors in `QueryBuilder` over and over:

```
[ERROR] 7005: com.datastax.driver.core.querybuilder.QueryBuilder: Parameter 1 of 'public com.datastax.driver.core.querybuilder.Clause gt(java.util.List, java.util.List)' has changed its type to java.lang.Iterable
[ERROR] 7005: com.datastax.driver.core.querybuilder.QueryBuilder: Parameter 2 of 'public com.datastax.driver.core.querybuilder.Clause gt(java.util.List, java.util.List)' has changed its type to java.lang.Iterable
[ERROR] 7005: com.datastax.driver.core.querybuilder.QueryBuilder: Parameter 1 of 'public com.datastax.driver.core.querybuilder.Clause gt(java.util.List, java.util.List)' has changed its type to java.lang.Iterable
[ERROR] 7005: com.datastax.driver.core.querybuilder.QueryBuilder: Parameter 2 of 'public com.datastax.driver.core.querybuilder.Clause gt(java.util.List, java.util.List)' has changed its type to java.lang.Iterable
[ERROR] 7005: com.datastax.driver.core.querybuilder.QueryBuilder: Parameter 1 of 'public com.datastax.driver.core.querybuilder.Clause gte(java.util.List, java.util.List)' has changed its type to java.lang.Iterable
[ERROR] 7005: com.datastax.driver.core.querybuilder.QueryBuilder: Parameter 2 of 'public com.datastax.driver.core.querybuilder.Clause gte(java.util.List, java.util.List)' has changed its type to java.lang.Iterable
[ERROR] 7005: com.datastax.driver.core.querybuilder.QueryBuilder: Parameter 1 of 'public com.datastax.driver.core.querybuilder.Clause gt(java.util.List, java.util.List)' has changed its type to java.lang.Iterable
[ERROR] 7005: com.datastax.driver.core.querybuilder.QueryBuilder: Parameter 2 of 'public com.datastax.driver.core.querybuilder.Clause gt(java.util.List, java.util.List)' has changed its type to java.lang.Iterable
[ERROR] 7005: com.datastax.driver.core.querybuilder.QueryBuilder: Parameter 1 of 'public com.datastax.driver.core.querybuilder.Clause gte(java.util.List, java.util.List)' has changed its type to java.lang.Iterable
[ERROR] 7005: com.datastax.driver.core.querybuilder.QueryBuilder: Parameter 2 of 'public com.datastax.driver.core.querybuilder.Clause gte(java.util.List, java.util.List)' has changed its type to java.lang.Iterable
[ERROR] 7005: com.datastax.driver.core.querybuilder.QueryBuilder: Parameter 1 of 'public com.datastax.driver.core.querybuilder.Clause gt(java.util.List, java.util.List)' has changed its type to java.lang.Iterable
[ERROR] 7005: com.datastax.driver.core.querybuilder.QueryBuilder: Parameter 2 of 'public com.datastax.driver.core.querybuilder.Clause gt(java.util.List, java.util.List)' has changed its type to java.lang.Iterable
[ERROR] 7005: com.datastax.driver.core.querybuilder.QueryBuilder: Parameter 1 of 'public com.datastax.driver.core.querybuilder.Clause gte(java.util.List, java.util.List)' has changed its type to java.lang.Iterable
[ERROR] 7005: com.datastax.driver.core.querybuilder.QueryBuilder: Parameter 2 of 'public com.datastax.driver.core.querybuilder.Clause gte(java.util.List, java.util.List)' has changed its type to java.lang.Iterable
```

While probably too coarse grained, I don't imagine this becoming much of an issue as I doubt we will make much changes in `QueryBuilder` on the 3.x branch in the future.